backend/Tasque.WebAPI/appsettings.Development.json

```diff
@@ -5,5 +5,11 @@
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning"
     }
+  },
+  "JwtIssuerOptions": {
+    "Issuer": "http://localhost:7143",
+    "Audience": "http://localhost:7143",
```

Comment:
Are you sure that the Audience should be on the same port as Issuer ?

Reply:
> Are you sure that the Audience should be on the same port as Issuer ?

As far as I know It's ok to use same port for both